### PR TITLE
Fix a small typo

### PR DESCRIPTION
**Pull request by moodmosaic**

### CODE REVIEW

**Comment by moodmosaic** [comment]
Great work, btw! :heart:

File: app/routes/__posts/00-annexes/03-coding-style.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ In order to ease the readability of the code, limit to 80 characters per line:
 
 ### One line function
 
-It ok to write functions on one line when they are short. IT's also ok to write theme on multiple lines if you find it more consistent.
+It ok to write functions on one line when they are short. IT's also ok to write them on multiple lines if you find it more consistent.
```
Comment (hugocaillard) [inline app/routes/__posts/00-annexes/03-coding-style.md:29]:
```suggestion
It's ok to write functions on one line when they are short. It's also ok to write them on multiple lines if you find it more consistent.
```

**Review by hugocaillard — CHANGES_REQUESTED** [review]
Hey @moodmosaic 
Thanks so much for the PR, I realised that this line was actually full of typos so made a quick review 👍

**Comment by moodmosaic** [comment]
Great! Included in this PR is also the suggestion you added. Should be good to go (I guess) :eyes:

**Review by hugocaillard — APPROVED** [review]

**Comment by hugocaillard** [comment]
Thanks a lot @moodmosaic. It makes you the first contributor on this project 😀

**Comment by moodmosaic** [comment]
:tada:

**Pull request merged**